content/en/authors/watroba/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+# Display name
+title: Marcin Wątroba
+
+# Username (this should match the folder name)
+authors:
+- watroba
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: PhD Student
+
+# Organizations/Affiliations
+organizations:
+- name: Department of Computational Intelligence, Wroclaw University of Science and Technology
+  url: ""
+
+# Short bio (displayed in user profile at end of posts)
+bio: My research interests include Natural Language processing, Deep Learning and Social Media Analysis.
+
+interests:
+- Social Media Analysis
+- Natural Language processing
+- Deep Learning
+
+education:
+  courses:
+  - course: MsC in Computer Science
+    institution: Wroclaw University of Science and Technology
+    year: 2020
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+#- icon: envelope
+#  icon_pack: fas
+#  link: '#contact'  # For a direct email link, use "mailto:test@example.org".
+- icon: github
+  icon_pack: fab
+  link: https://github.com/markowanga
+# Link to a PDF of your resume/CV from the About widget.
+# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
+# - icon: cv
+#   icon_pack: ai
+#   link: files/cv.pdf
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+email: "marcin.watroba@pwr.edu.pl"
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- PhD Students
```

Comment:
your are very welcome, but for now comment this line :)

Reply:
Ok

Reply:
Haha 😆